fc-community-extension/src/contentModifiers.js

```diff
@@ -47,6 +47,13 @@ export const alterRatingPageTwitterNote = (twitterNote) => {
         noteUrl,
         content,
       });
+      logger.log("Got res for note", res);
+      const { transaction, error } = await mintXNote(res.id, 1, res.hash, res.signature);
+      if (error) {
+        logger.log("Failed to mint note", error);
+      } else {
+        logger.log(`Success! ${transaction}`);
+      }
```

Comment:
Hm looks like you did a bad rebase/merge? This does not exist anymore on main and should be removed.

Reply:
indeed, I completely missed my rebase